pages/profile/[username].tsx

```diff
@@ -0,0 +1,165 @@
+                                        <Image src='/image/Icon_Follow.svg' alt='follow icon' width={15.81} height={20} />
+                                    </button>
+                                    <button className="border border-gray02 h-10 w-10 rounded-lg flex justify-center items-center" onClick={() => onShareButtonClicked()}>
+                                        <Image src='/image/Icon_Share.svg' alt='share icon' width={20} height={20} />
+                                    </button>
+                                </div>
+                            </div>
+                            {/* Horizontal Devider */}
+                            <div className="border-t border-gray02" />
+                            {/* Card Content */}
+                            <div>
+                                <text className="font-normal text-sm">{data?.bio ?? ''}</text>
+                            </div>
+                        </div>
+                    </LoaderWrapper>
+                </div>
+                {/* Input Message */}
+                {!isLoading && <div className={data?.allow_anon_dm ? "w-full bg-white md:max-w-M lg:max-w-M xl:max-w-M p-2 fixed bottom-0 flex flex-row gap-[6px] z-[9999]" : "max-w-[375px] p-2 mb-4 bg-gray05 fixed bottom-0 flex flex-row gap-[6px] rounded-lg z-[9999]"}>
+                    <Image className="rounded-full" src='/image/anonIcon.svg' alt="anon icon" width={24} height={24} />
+                    {data?.allow_anon_dm ? (
+                        <>
+                            <div className="flex flex-grow items-ce bg-gray05 rounded-xl py-1 px-2">
+                                <textarea ref={textAreaRef} rows={1} onChange={(e) => {
+                                    setMessage(e.target.value)
+                                }} className="bg-transparent min-h-[24px] w-full"
+                                    style={{ resize: 'none' }}
+                                    placeholder="Send a message..." />
+                                <Toggle
+                                    checked={true}
+                                    className="bg-white"
+                                    defaultChecked={true}
+                                    onClick={() => showDownloadToast()}
+                                    icons={{
+                                        checked: <text className="text-xs font-normal text-cyan leading-[18px]">On</text>,
+                                        unchecked: null,
+                                    }}
+                                />
+                            </div>
+                            <button className="rounded-full flex-shrink-0 bg-cyan h-8 w-8 flex items-center justify-center" onClick={() => {
+                                localStorage.setItem(MessageEnum.tempMessage, message);
+                                router.push('/verification')
+                            }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment about local storage, storing the temporary message in local storage might not be the best approach. If you need to pass this data to another page, consider using state management libraries like Redux or context API, or passing as parameters using router.

```diff
- localStorage.setItem(MessageEnum.tempMessage, message);
- router.push('/verification')
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->